app/src/main/java/com/zeapo/pwdstore/autofill/oreo/ui/AutofillFilterActivity.kt

```diff
@@ -162,6 +162,12 @@ class AutofillFilterView : AppCompatActivity() {
                 setText(initialSearch, TextView.BufferType.EDITABLE)
                 addTextChangedListener { updateSearch() }
             }
+            origin.text = buildSpannedString {
+                append("Search and fill into\n")
```

Comment:
Expose for localisation?

Reply:
Should be done, added a German translation for good measure